### PR TITLE
Reorganize Media Gallery stores configuration UI

### LINKED ISSUE #1738: Reorganize Media Gallery stores configuration UI
### Steps to reproduce (*)
1. Login to Admin Panel
2. Go to Stores -> Configuration -> Advanced -> System

### Expected result (*)
0. **The changes should be applied to the UI only, the values or configuration path should not be affected**
1. Section label is changed from `Enhanced Media Gallery` to `Media Gallery`
2. Select label is changed from `Enable` to `Enable Old Media Gallery`
3. Select option values should be reversed
4. Corresponding MFTF action groups should be corrected
image

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will reversed the media gallery configuration

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1738 : Reorganize Media Gallery stores configuration

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by konarshankar07** [comment]
@magento run all tests

File: app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml [hunk 1]
```diff
@@ -7,7 +7,21 @@
...
-        <add id="Magento_MediaGalleryUi::media" title="Media" translate="title" module="Magento_MediaGalleryUi" sortOrder="15" parent="Magento_Backend::content" resource="Magento_Cms::media_gallery" dependsOnConfig="system/media_gallery/enabled"/>
-        <add id="Magento_MediaGalleryUi::media_gallery" title="Media Gallery" translate="title" module="Magento_MediaGalleryUi" sortOrder="0" parent="Magento_MediaGalleryUi::media" action="media_gallery/media/index" resource="Magento_Cms::media_gallery"/>
+        <add id="Magento_MediaGalleryUi::media"
+             title="Media"
+             translate="title"
+             module="Magento_MediaGalleryUi"
+             sortOrder="15"
+             parent="Magento_Backend::content"
+             resource="Magento_Cms::media_gallery"
+             dependsOnConfig="system/media_gallery/enabled"/>
```
Comment (konarshankar07) [inline app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml]: @sivaschenko ...
Do we need to display this menu if old media gallery is enabled?
Thanks

**Comment by konarshankar07** [comment]
@magento run Integration Tests

**Comment by konarshankar07** [comment]
@magento run Functional Tests CE

**Comment by konarshankar07** [comment]
@magento run Functional Tests CE

File: app/code/Magento/MediaGalleryUi/etc/adminhtml/system.xml [hunk 2]
```diff
@@ -9,9 +9,9 @@
     <system>
         <section id="system">
             <group id="media_gallery" translate="label" type="text" sortOrder="1000" showInDefault="1" showInWebsite="0" showInStore="0">
-                <label>Enhanced Media Gallery</label>
+                <label>Media Gallery</label>
                 <field id="enabled" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
-                    <label>Enabled</label>
+                    <label>Enable Old Media Gallery</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryUi/etc/adminhtml/system.xml]: @konarshankar07 can you please keep the existing value for enabled media gallery and just change the option values for Stores Configuration.

File: app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml (hunk 1 above)
Comment (sivaschenko) [reply app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml]: Only Sores configuration representation should be changed in scope of the issue

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PRs @konarshankar07 ! Please see my comments

File: app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml (hunk 1 above)
Comment (konarshankar07) [reply app/code/Magento/MediaGalleryUi/etc/adminhtml/menu.xml]: But I didn't understood if `Enable Old Media Gallery` is set to Yes then the menu will be visible

**Comment by konarshankar07** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-8014](https://jira.corp.magento.com/browse/ENGCOM-8014) has been created to process this Pull Request

**Comment by engcom-Lima** [comment]
:heavy_check_mark: QA Passed
1. Section label is changed from `Enhanced Media Gallery` to `Media Gallery`
2. Select label is changed from `Enable` to `Enable Old Media Gallery`
3. Select option values should be reversed
Screenshot_116

**Pull request merged**